src/qt/buynamespage.cpp

```diff
@@ -72,12 +111,83 @@ void BuyNamesPage::onNameEdited(const QString &name)
     }
 }
 
+void BuyNamesPage::onAsciiNameEdited(const QString &name) {
+    if (!walletModel)
+        return;
+
+    try
+    {
+        const QString hexName = ASCIIToHex(name);
+        const QString domainName = ASCIIToDomain(name);
+
+        ui->registerNameHex->setText(hexName);
+        ui->registerNameDomain->setText(domainName);
+
+        RefreshAvailableError();
+    }
+    catch(InvalidNameString &e)
+    {
+        ui->statusLabel->setText(tr("Not a valid ASCII entry!"));
+    }
+
+}
+
+void BuyNamesPage::onHexNameEdited(const QString &name) {
+    if (!walletModel)
+        return;
+
+    try{
```

Comment:
Please either put the brace on a separate line, or add a space between `try` and the brace.